src/datasources/floodscan.py

```diff
@@ -52,10 +53,20 @@ def load_floodscan_cogs(
         da_in = process_floodscan_cog(
             cog_name=cog, container_name="global", mode="dev"
         )
-        das.append(da_in)
+        da_sfed = subset_band(da_in, band="SFED")
```

Comment:
Nice way to do it!

Reply:
Yes I much prefer this to my janky method 😆